tools/unitctl/Cargo.lock

```diff
@@ -355,6 +439,16 @@ version = "1.9.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4f8a51dd197fa6ba5b4dc98a990a43cc13693c23eb0089ebb0fcc1f04152bca6"
 
+[[package]]
+name = "deranged"
```

Comment:
Heh, just have to ask what this is!?

Reply:
It looks like at some point some rust dev wanted some fancy integers that were specified and optimized to fit within exact ranges. 
https://docs.rs/deranged/0.3.11/deranged/

Reply:
Heh, OK.